content/en/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -0,0 +1,319 @@
+---
+title: Apply Pod Security Standards at the Cluster Level
+content_type: tutorial
+weight: 10
+---
+
+{{% alert title="Note" %}}
+This tutorial applies only for new clusters.
+{{% /alert %}}
+
+Pod Security admission (PSA) is enabled by default in v1.23 and later, as it [graduated
+to beta](/blog/2021/12/15/pod-security-admission-beta/). Pod Security Admission
```

Comment:
https://github.com/kubernetes/website/pull/30502 has the article due for publication on 2021-12-09

```suggestion
Pod Security admission (PSA) is enabled by default in v1.23 and later, as it has
[graduated to beta](/blog/2021/12/09/pod-security-admission-beta/).
Pod Security 
```